core/src/sumeragi/mod.rs

```diff
@@ -366,7 +366,9 @@ impl<G: GenesisNetworkTrait, K: KuraTrait, W: WorldTrait, F: FaultInjection>
             return;
         }
         let txs = self.queue.get_transactions_for_block();
-        if let Err(error) = self.round(txs, ctx).await {
+        // TODO: This should properly process triggers
+        let trigger_recommendations = Vec::new();
```

Comment:
From what this PR is suggesting,
`wsv.triggers.recommendations` works as a temporary store of the recommendations
which is based on events emitted by previous block application, right?
Then Sumeragi somehow can receive the recommendation from WSV

Reply:
Exactly. 

I'm wondering if there's more kinds of validation that we could do.

Reply:
Could you create an issue for this TODO?